test/coverage.config

```diff
@@ -3,7 +3,14 @@
   <CodeCoverage>
     <ModulePaths mergeDefaults="False">
       <Include>
+        <ModulePath>.*Microsoft\.Testing\.Extensions\.CrashDump\.dll$</ModulePath>
```

Comment:
@jakubch1 @MarcoRossignoli could you please validate this config file?!

Reply:
it looks good